ui/app/AppLayouts/Wallet/popups/ReceiveModal.qml

```diff
@@ -41,6 +41,15 @@ StatusModal {
     showHeader: false
     showAdvancedHeader: true
 
+    // When no network is selected reset the prefix to empty string
+    Connections {
+        target: RootStore.enabledNetworks
+        onModelReset: {
```

Comment:
Please use the syntax `function onFooChanged(params)`; this one generates tons of warnings